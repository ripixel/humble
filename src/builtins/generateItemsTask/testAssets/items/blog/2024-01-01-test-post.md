This is a test post.

Second paragraph for excerpt.
