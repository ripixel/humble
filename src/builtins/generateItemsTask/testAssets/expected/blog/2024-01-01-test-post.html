<html>
  <body>
    <header>
      <h1>Title from header: Test Post</h1>
    </header>
    <main>
      <h2>Title from template: Test Post</h2>
      <article>
        <p>This is a test post.</p>
        <p>Second paragraph for excerpt.</p>
      </article>
      <footer>Published: 1 January 2024</footer>
    </main>
    <footer>Copyright Test</footer>
  </body>
</html>
